src/views/products/ProductForm.vue

```diff
@@ -65,25 +91,33 @@ import Checkbox from 'primevue/checkbox'
 import InputUnit from '@/components/InputUnit.vue'
 import InputSupplier from '@/components/InputSupplier.vue'
 import InputCategory from '@/components/InputCategory.vue'
+import Calendar from 'primevue/calendar';
+import ProductsPriceHistory from './ProductsPriceHistory.vue'
+
 
 export default {
   components: {
-    InputUnit, InputSupplier, InputNumber, InputCategory, Divider, Checkbox,
+    InputUnit, InputSupplier, InputNumber, InputCategory, Divider, Checkbox, Calendar, ProductsPriceHistory,
   },
   data() {
     return {
       visible: false,
       loading: false,
       product: {},
+      productPriceValue: null,
     }
   },
   methods: {
     show(object = {}) {
       this.product = { ...object }
       this.visible = true
+      this.productPriceValue = this.getLastPriceValue(this.product)
     },
     async saveProduct() {
       if (this.product.name) {
+        // prepare product.prices
+        this.savePrice()
```

Comment:
If calling the method need a comment, it's likely than the method name is not good :) maybe rename it to preparePricesBeforeSave or something like this so it's self explanatory